.github/workflows/build.yml

```diff
@@ -19,19 +19,15 @@ jobs:
     tests:
         runs-on: ubuntu-18.04
 
-        name: "PHP ${{ matrix.php }}, MySQL ${{ matrix.mysql }}"
+        name: "PHP ${{ matrix.php }}, MySQL ${{ matrix.mysql }}, Sylius ${{ matrix.sylius }}"
 
         strategy:
             fail-fast: false
             matrix:
-                php: [7.4, 7.3]
+                php: [8.0, 8.1]
```

Comment:
we should test 7.4 and 8.1

Reply:
In fact we should change the required PHP version to ^8.0 a 7.4 is no longer supported in Sylius 1.11 💃